lq1/default.cfg

```diff
@@ -58,13 +59,44 @@ bind	"-"		"sizedown"
 +speed
 
 //Cvars
-gl_texturemode "GL_NEAREST_MIPMAP_LINEAR"
-gl_texture_anisotropy "8"
-registered "1""
-vid_fsaamode "1"
-vid_fsaa "4"
-volume "0.7""
-
+gl_texturemode gl_nearest_mipmap_linear
+gl_texturemode_sky gl_nearest
+gl_texturemode_hud gl_nearest
+gl_texturemode2d gl_nearest
+gl_overbright 1
+gl_overbright_model 1
+gl_smoothfont 0
+gl_texture_anisotropy 16
+vid_width "1280"
+vid_height "800
+vid_fsaamode 0
+vid_fsaa 8

```

Comment:
Forcing 8x FSAA on certain sourceports is a serious performance concern.

Reply:
understood will remove

Reply:
I will mark this as resolved under the understanding that if users complain about performance issues, this be the first item to investigate and consider removing.

Reply:
understood